recipes-debian/gpgme/gpgme_debian.bb

```diff
@@ -0,0 +1,90 @@
+SUMMARY = "High-level GnuPG encryption/signing API"
```

Comment:
Could you add header description if you refered to poky's recipe?

Reply:
I see